README.md

```diff
@@ -126,375 +126,122 @@ If your car is not listed above, then there has been no documented information o
 * [Key Extraction](#key-extraction) if you don't know the key.
 * [Key Installation](#key-installation) if you know the key.
 
-> [!NOTE]
-> The key will change if you get a new bumper, because the bumper has distance sensors that use the security key. Instead of applying the existing key to the bumper, they replace the key on all parts of the car. The same goes for many other parts with secoc components.
->
-> If you never get into an accident, then the key will never change, unless a Toyota service technician presses a wrong button.
-
-
 ## Key Extraction
 
 Your car has a security key that Toyota doesn't want you to have. \
 Follow this guide to run a [hardware exploit](https://icanhack.nl/blog/secoc-key-extraction/) to extract the key.
 
-### Step 1. Upgrade AGNOS by installing `commaai/nightly-dev`
+### Step 1. Install `TSK Manager`
 
-AGNOS is the operating system used in C3X. The latest one is needed to run TSK Manager.
+At home, turn on C3X with your phone charger. Ignore the low voltage warning.
 
-1-1. At home, turn on C3X with your phone charger. Ignore the low voltage warning. USB A-to-C cables work well, and USB PD (Power Delivery) sometimes doesn't work. If all fails, you can do this in your car.
+Don't choose `Install openpilot`. Instead, choose `Custom Software` with URL `tsk.lvin.ca`
```

Comment:
Do we need the "Don't choose Install openpilot"? Saying "On boot tap 'Custom Software'" would suffice

Reply:
Changed to

Choose `Custom Software` and enter the URL `tsk.lvin.ca`